client/src/Component/Main/Main.jsx

```diff
@@ -0,0 +1,72 @@
+import "./main.scss";
+// import Data from "../../data/exampleresponse.json";
+import VideoCard from "./VideoCard";
+import AddVideo from "./AddVideo";
+import { useState, useEffect } from "react";
+
+const Videos = () => {
+  const [videosData, setVideoData] = useState([]);
+
+  // const addVideo = (formValue) => {
+  //   setVideoData([...videosData, formValue]);
+  // };
+
+  useEffect(() => {
+    async function fetchData() {
+      console.log("hello");
+      try {
+        const response = await fetch("http://localhost:5000");
+        const data = await response.json();
+        setVideoData(data.video);
+      } catch (error) {
+        console.error(error);
+      }
+    }
+
+    fetchData();
+  }, []);
+
+  const addVideo = async (input) => {
+    try {
+      const response = await fetch("http://localhost:5000", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(input),
+      });
+
+      const data = await response.json();
+
+      if (data.result) {
+        setVideoData(data.video);
+      } else {
+      }
+    } catch {}
```

Comment:
Nice, catching the errors! And using async/await!

A good extension would be to tell the user something went wrong, by doing something with the catch.

Reply:
i jest added for now console.log(error) but I am going to set a text on the page to show if the video posted or not